cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskManager.java

```diff
@@ -321,11 +323,82 @@ private void completeTask(ExecutionTask task) {
     }
   }
 
-  public void clear() {
+  public synchronized void clear() {
     _brokersToSkipConcurrencyCheck.clear();
     _inProgressReplicaMovementsByBrokerId.clear();
     _inProgressPartitions.clear();
     _executionTaskPlanner.clear();
     _executionTaskTracker.clear();
   }
+
+  public synchronized ExecutionState getExecutionState() {
```

Comment:
Nit: we have already got many "State". Can we have something like ExecutionTasksSummary here? Also the convention in Kafka, as well as KafkaCruiseControl, is to avoid using "get" in the getter method.

Reply:
Renamed: `getExecutionState` -> `getExecutionTasksSummary`.
Since this method does not just return an existing object `x`, but building one within this method, it is named `getX` instead of `x`.